src/main/angular/constants.ts

```diff
@@ -0,0 +1,21 @@
+/** Contains several constants in use throughout the application. */
+/** Whether to use test data or not. */
+export const USE_TEST_DATA = true;
+
+// See https://standards.google/guidelines/brand-standards/color/palette/#brand-colors
+/** Default colors for use with the projects. */
+export const DEFAULT_COLORS: string[] = [
+  // Google blue
+  '#174ea6',
```

Comment:
consider making these consts themselves. you'll probably need them more than once 

e.g 
const GOOGLE_RED='';
...

then DEFAULT_COLORS = string[]  = [GOOGLE_RED,...];